src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -554,6 +557,16 @@ public void apply(VariantContext vc, ReadsContext readsContext, ReferenceContext
                 return;
         }
 
+        // Initialize PL index to allele indices for all samples
```

Comment:
Changed to `Initialize the cache of PL index to a list of alleles for each ploidy.`